ci/benchmark_parser.py

```diff
@@ -37,16 +39,19 @@
                     help='Check for results in subdirectories')
 parser.add_argument('--key-sizes', dest='key_sizes', action='store_true',
                     help='Parse only the results regarding keys size measurments')
+parser.add_argument('--ops-per-dollar', dest='ops_per_dollar', action='store_true',
+                    help='Compute and append number of operations per dollar')
```

Comment:
I don't see the argument being used when launching the benches ?

Reply:
It is used only for PBS benchmark for now. Is it meaningful to perform such computation on boolean and shortint benchmarks also ? What do you think @jborfila ?

Reply:
ah no you are right, I missed it